apisix/plugins/kafka-logger.lua

```diff
@@ -83,6 +83,11 @@ local schema = {
         -- in lua-resty-kafka, cluster_name is defined as number
         -- see https://github.com/doujiang24/lua-resty-kafka#new-1
         cluster_name = {type = "integer", minimum = 1, default = 1},
+        -- config for lua-resty-kafka, default value is same as lua-resty-kafka
+        kafka_batch_num = {type = "integer", minimum = 1, default = 200},
```

Comment:
Would be better to use `producer` prefix instead of `kafka`? Since this plugin is already a kafka logger.

Reply:
good idea,i will change it